challenges/ecosystem/05.md

```diff
@@ -215,3 +215,39 @@ Now you can run `npm run test:watch` to watch your files and run tests automatic
 
 - https://reactnativetesting.io/
 - https://github.com/vanGalilea/react-native-testing?tab=readme-ov-file#covered-examples-
+
+
+### 👽 Bonus
```

Comment:
I would create a `ecosystem/06.md` challenge here because maestro is end-to-end. It's more high level and different to unit testing.